src/components/Form.vue

```diff
@@ -45,14 +45,31 @@
     </b-modal>
     <div class="p-5">
       <h1 class="py-3">Filled Details</h1>
-      <b-table hover :items="items" :fields="fields">
-        <template v-slot:cell(edit)="row">
-          <a @click="edit(row)">Edit</a>
-        </template>
-        <template v-slot:cell(delete)="row">
-          <a @click="del(row)">Delete</a>
-        </template>
-      </b-table>
+      <table class="table table-bordered">
+        <thead class="bg-light">
+          <tr>
```

Comment:
can you use loop here

Reply:
no, if i use loop  here ,i cannot apply event on only  edit here

Reply:
this is bad practice **items[0]** if we have 100's of items this wont be feasible